### DISCUSSION TITLE
have created ConfigMap for keda-http-routing-table but while running the interceptor getting error

### DISCUSSION THREAD

**Discussion by AAnshuman19** [kedacore/http-add-on#1203] [General]
ConfigMap 

Name:         keda-http-routing-table
Namespace:    keda
Labels:       app=http-add-on
              control-plane=operator
              keda.sh/addon=http-add-on
              name=http-add-on-routing-table
Annotations:  <none>

Data
====
keda-test:
----
keda-config

BinaryData
====

Events:  <none>

Error 

{"level":"info","ts":1733736346.2083926,"caller":"interceptor/main.go:44","msg":"starting interceptor","timeoutConfig":{"Connect":500000000,"KeepAlive":1000000000,"ResponseHeader":500000000,"DeploymentReplicas":20000000000,"ForceHTTP2":false,"MaxIdleConns":100,"IdleConnTimeout":90000000000,"TLSHandshakeTimeout":10000000000,"ExpectContinueTimeout":1000000000},"servingConfig":{"CurrentNamespace":"keda","ProxyPort":8080,"AdminPort":9090,"ConfigMapCacheRsyncPeriod":3600000000000,"DeploymentCachePollIntervalMS":250}}
W1209 09:25:46.209575       1 shared_informer.go:498] resyncPeriod 250ms is too small. Changing it to the minimum allowed value of 1s
{"level":"info","ts":1733736346.2096305,"caller":"interceptor/main.go:79","msg":"Interceptor starting"}
{"level":"info","ts":1733736346.2096717,"caller":"interceptor/main.go:93","msg":"Fetching initial routing table"}
{"level":"error","ts":1733736346.221434,"logger":"pkg.routing.GetTable","caller":"routing/config_map.go:98","msg":"failed decoding routing table ConfigMap","configMapName":"keda-http-routing-table","error":"no 'routing-table' key found in the keda-http-routing-table ConfigMap","stacktrace":"github.com/kedacore/http-add-on/pkg/routing.GetTable\n\t/go/src/github.com/kedacore/http-add-on/pkg/routing/config_map.go:98\nmain.main\n\t/go/src/github.com/kedacore/http-add-on/interceptor/main.go:96\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:255"}
{"level":"error","ts":1733736346.221491,"caller":"interceptor/main.go:103","msg":"fetching routing table","error":"failed decoding ConfigMap keda-http-routing-table into a routing table: no 'routing-table' key found in the keda-http-routing-table ConfigMap","errorVerbose":"no 'routing-table' key found in the keda-http-routing-table ConfigMap\nfailed decoding ConfigMap keda-http-routing-table into a routing table\ngithub.com/kedacore/http-add-on/pkg/routing.GetTable\n\t/go/src/github.com/kedacore/http-add-on/pkg/routing/config_map.go:104\nmain.main\n\t/go/src/github.com/kedacore/http-add-on/interceptor/main.go:96\nruntime.main\n\t/usr/local/go/src/runtime/proc.go:255\nruntime.goexit\n\t/usr/local/go/src/runtime/asm_amd64.s:1581","stacktrace":"main.main\n\t/go/src/github.com/kedacore/http-add-on/interceptor

**Comment by linkvt** [comment]
Closing this discussion as I'm doing a cleanup right now, if this is a bug that still exists please create an issue, otherwise just let us know.
